src/views/ResultsPage/PrintModalContent.jsx

```diff
@@ -124,5 +121,6 @@ PrintModalContent.propTypes = {
 	selectedList: PropTypes.array,
 	handleClose: PropTypes.func,
```

Comment:
We have `handleClose` and `closeModal` which I believe should be doing the same thing, but they aren't even used in this component. Know it's not a change you made, just raising the question.

Reply:
I agree with this one - 	handleClose  is being passed into component from Res page but it is not used in component itself.. I did not touch it as part of this ticket, unless this is a bad code and we want to fix it up

Reply:
You can go ahead and remove from proptypes as it isn't being handled.  This is probably artifact from a previous implementation.

Reply:
It is gone @arcepaul